src/machine/behavior/is/control/jmp.rs

```diff
@@ -1,6 +1,15 @@
 use machine::state::State;
 
-pub fn jmp(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn jmp(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let x: u64 = (x as u64) << 16;
+    let y: u64 = (y as u64) <<  8;
```

Comment:
These operations will work but they are not really `// Load operands`. `// Execute` would fit better.